components/lib/crash/README.md

```diff
@@ -82,13 +82,29 @@ Add a `MozillaSocorroService` instance to your `CrashReporter` in order to uploa
 ```Kotlin
 CrashReporter(
     services = listOf(
-		MozillaSocorroService(applicationContext, "your app name"),
+		MozillaSocorroService(applicationContext, "your app name")
     )
 ).install(applicationContext)
 ```
 
 ⚠️ Note: Currently only native code crashes get uploaded to Socorro. Socorro has limited support for "uncaught exception" crashes too, but it is recommended to use a more elaborate solution like Sentry for that.
 
+### Sending crash reports to Glean
+
+[Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html) is a new way to collect telemetry by Mozilla.
+
+Add a `GleanCrashReporterService` instance to your `CrashReporter` in order to record crashes in Glean:
```

Comment:
I'm missing documentation here on:
- What kind of information is recorded into the Glean SDK.
- How the data flows from the crash process to get recorded into the Glean SDK.

Is this documented anywhere else (outside of code)?

Reply:
No, nothing outside of code.  I was planning on adding a section to the documentation explaining this but with the focus on getting experiments done first has kept me from getting back to this PR.

Reply:
I added to the `README.md`, let me know if we want to add anything.  If we want to document this elsewhere, I'll file a follow-up bug.